git-server/src/error.rs

```diff
@@ -146,4 +155,10 @@ impl Error {
     }
 }
 
-impl warp::reject::Reject for Error {}
+impl IntoResponse for Error {
```

Comment:
I didn't use `recover` implementation since `self` is an `Error` and has no `is_not_found()` method. This `IntoResponse` only returns status codes defined above in `pub fn status(&self)`.